example/django/README.md

```diff
@@ -1,69 +1,44 @@
 # Bugsnag Django demo
 
-This Django application demonstrates how to use Bugsnag with the Django web
-framework for Python. Before testing it, open up the `bugsnag_demo/settings.py`
-file and configure your API key (see the very bottom of the file).
+This Django application demonstrates how to use Bugsnag with the Django web framework for Python.
 
-```
-BUGSNAG = {
-    "api_key": "066f1ad3590596f9aacd601ea89af845"
-}
-```
+Please note this particular example app will only work with Python 3+ and Django 2+, to showcase the most current configurations. Check out [this example](https://github.com/bugsnag/bugsnag-python/example/django1_10) to see the configuration for  Python 2 and Django 1.
 
-Install dependencies.
+## Setup
 
-```
-pip install -r requirements.txt
-```
+Try this out with [your own Bugsnag account](https://app.bugsnag.com/user/new)! You'll be able to see how the errors are reported in the dashboard, how breadcrumbs are left, how errors are grouped and how they relate to the original source.
 
-This minimal setup is enough to start Bugsnagging.
+To get set up, follow the instructions below. Don't forget to replace the placeholder API token with your own!
 
-Run the application.
 
-```
-python manage.py runserver
-```
+1. Clone the repo and `cd` into this directory:
+    ```shell
+    git clone https://github.com:bugsnag/bugsnag-python.git
+    cd bugsnag-python/example/django
+    ```
 
-Next, open your project's dashboard on Bugsnag.
+1. Install dependencies
+    ```shell
+    pip install -r requirements.txt
+    ```
 
-1. [crash](/crash)
-<br/>
-Crashes the library and sends a notification about the nature of the crash.
-Basically, almost any unhandled exception sends a notification to Bugsnag.
-Pressing this link would lead to an empty page, which is normal.
-See `demo/views.py, #crash`
+1. Before testing it, open up the [`settings.py`](bugsnag_demo/settings.py)
+    file and configure your API key.
 
-1. [crash and use callbacks](/crash_with_callback)
-<br/>
-Before crashing, the library would append the Diagnostics tab with some
-predefined information, attached by means of a callback.
-Pressing this link would lead to an empty page, which is normal.
-See `demo/views.py, #crash_with_callback`
+1. Confirm that Bugsnag middleware is added to the top of MIDDLEWARE in [`settings.py`](bugsnag_demo/settings.py)
+    ```shell
+    MIDDLEWARE = (
+    'bugsnag.django.middleware.BugsnagMiddleware',
+    ...
+    )
+    ```
 
-1. [notify](/notify)
-<br/>
-Bugsnag Python provides a way to send notifications on demand by means of
-`bugsnag.notify()`. This API allows to send notifications manually, without
-crashing your code.  See `demo/views.py, #notify`
+1. Run the application. (Make sure to use any 3+ version of python.)
+    ```shell
+    python manage.py runserver
+    ```
```

Comment:
Running this on my machine resulted in the following message in red text:

```
You have 13 unapplied migration(s). Your project may not work properly until you apply the migrations for app(s): admin, auth, contenttypes, sessions.
Run 'python manage.py migrate' to apply them.
```

Is this expected?